packages/runtime/datastore-definitions/src/channel.ts

```diff
@@ -91,6 +92,7 @@ export interface IChannel extends IFluidLoadable {
 		fullTree?: boolean,
 		trackState?: boolean,
 		telemetryContext?: ITelemetryContext,
+		incrementalContext?: IIncrementalContext,
```

Comment:
This makes the `summarize` method inconsitent across the layers. This should be added to the summarize method in data stores and container runtime as well to keep it consistent.
We had worked hard in the past to make them consistent and we should ensure they still are :-).

Reply:
My opinion is that this is the minimal change required on our end. To make summarize method consistent, that would be another refactoring. I had done that in other PRs, but then this PR would be much harder to read.

Reply:
I will add a few tasks to make this consistent across summarize

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/3881